meson.build

```diff
@@ -0,0 +1,285 @@
+project('opus', 'c',
+  version: '1.3.1',
+  license : 'bsd-style',
```

Comment:
Thank you for adding a license, but it seems like you could be much more specific and declare it as the "BSD-3-Clause" license rather than the weaker "something like a BSD-ish" license.

https://spdx.org/licenses/BSD-3-Clause.html

Reply:
To keep things simple I'll release the code under MIT license, which the auto-generated `LICENSE.build` already presumes. I updated the entry in `meson.build` accordingly.

Reply:
Hmm? The license it's referring to is the license of the opus project, not the license of the meson.build files. opus is licensed under the BSD-3-Clause.

Reply:
Ah, of course. Sorry it's getting late :) Fixed.